modules/stochastic_tools/include/utils/GaussianProcessHandler.h

```diff
@@ -153,13 +156,19 @@ class GaussianProcessHandler
                            unsigned int iter,
                            const unsigned int & batch_size,
                            const Real & learning_rate,
-                           const bool & verbose);
+                           const bool & verbose,
+                           const Real & p_std);
```

Comment:
I would switch the order of these two arguments. Debugging or verbosity arguments are generally last.